### ISSUE TITLE
Unable to use sensu-install on Windows - embedded bin not added to path

### ISSUE THREAD

**Issue by AirCombat** [sensu/sensu-build#157]
C:\opt\sensu\bin\sensu-install.bat -p sensu-plugins-windows:0.0.10

[SENSU-INSTALL] installing Sensu plugins ...
[SENSU-INSTALL] provided Sensu plugins: ["sensu-plugins-windows:0.0.10"]
[SENSU-INSTALL] compiled Sensu plugin gems: ["sensu-plugins-windows:0.0.10"]
[SENSU-INSTALL] determining if Sensu plugin gem 'sensu-plugins-windows:0.0.10' is already installed ...
[SENSU-INSTALL] gem list -i sensu-plugins-windows --version '0.0.10'
[SENSU-INSTALL] Sensu plugin gem 'sensu-plugins-windows' has not been installed
[SENSU-INSTALL] Sensu plugin gems to be installed: ["sensu-plugins-windows:0.0.10"]
[SENSU-INSTALL] installing Sensu plugin gem 'sensu-plugins-windows:0.0.10'
[SENSU-INSTALL] gem install sensu-plugins-windows --version '0.0.10' --no-ri --no-rdoc --verbose
[SENSU-INSTALL] failed to install Sensu plugin gem 'sensu-plugins-windows'
[SENSU-INSTALL] please take note of any failure messages above
[SENSU-INSTALL] make sure you have build tools installed (e.g. gcc)
[SENSU-INSTALL] trying to determine the Sensu plugin homepage for sensu-plugins-windows ...
'gem' is not recognized as an internal or external command,
operable program or batch file.

C:\opt\sensu\embedded\bin should be added to %PATH% as part of the MSI

Why is mkdir conf.d not in the MSI either!!!??? :)

To Resolve:
$oldPath=(Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).Path
$newPath=$oldPath+’;C:\opt\sensu\embedded\bin\’
Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH –Value $newPath

**Comment by wbrodie** [comment]
I also forgot about the fact i use pathmunge in RHEL. This should be included in the rpm or a note in the documentation otherwise sensu-install etc wont work

**Comment by portertech** [comment]
We should add the embedded bin dir to the path in https://github.com/sensu/sensu-build/blob/master/assets/msi/files/sensu-install.bat

**Comment by wbrodie** [comment]
@portertech I would love it if the linux installer did it as well... pathmunging is a lot simpler on linux so isn't a biggy. :)

**Comment by pablolibo** [comment]
Hi guys, I can Fix it with this way:

From Powershell command:

```
PS C:\opt\sensu\embedded\bin> .\gem install sensu-plugins-windows
Fetching: sensu-plugins-windows-0.0.10.gem (100%)
You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu
Successfully installed sensu-plugins-windows-0.0.10
Parsing documentation for sensu-plugins-windows-0.0.10
Installing ri documentation for sensu-plugins-windows-0.0.10
1 gem installed
```

**Comment by amdprophet** [comment]
This is fixed in our new repository by the following commit: https://github.com/sensu/sensu-omnibus/commit/93ded1109f26b48ee0d3d5d1e4a6d2769b37fb1a and will be available in the Sensu 0.27 release.

**Comment by AirCombat** [comment]
When is .27 coming out?
